src/desktop.c

```diff
@@ -68,7 +28,7 @@ desktop_move_to_front(struct view *view)
 	}
 	move_to_front(view);
 #if HAVE_XWAYLAND
-	move_xwayland_sub_views_to_front(view);
+	xwayland_move_sub_views_to_front(view, move_to_front);
```

Comment:
Usually I am not that much a fan of function pointers for cases like this but this still seems cleaner than to make the `move_to_front()` helper public or to repeat its code in xwayland.c.

Reply:
Yeah it's a little bit awkward, but I was struggling to come up with a better alternative.

Reply:
I can't think of a better solution either so I guess this is it for now.